apps/ledger-live-desktop/src/renderer/components/SideBar/SideBarList.tsx

```diff
@@ -60,7 +60,7 @@ class SideBarList extends Component<Props> {
             collapsed={collapsed}
             scroll={scroll}
             flow={2}
-            px={3}
+            px={"12px"}
```

Comment:
You can use spacing from DS :)

https://react-ui-storybook.vercel.app/?path=/story/particles--spacing

`12px === px={5}`